invenio_pages/services/schemas.py

```diff
@@ -45,5 +45,6 @@ class PageSchema(Schema):
     content = DynamicSanitizedHTML()
     description = fields.String()
     template_name = fields.String()
+    has_custom_view = fields.Boolean()
```

Comment:
right now it is not customizable or viewable from the admin panel, since we did it for just one community, though there might be a need to add it in the future